boards/arm/rddrone_fmuk66/Kconfig.defconfig

```diff
@@ -20,6 +20,9 @@ config MCG_VDIV0
 config MCG_FCRDIV
 	default 1
 
+config I2C
+	default y
+
```

Comment:
This change shouldn't be necessary now that sensor drivers select I2C

Reply:
Acknowledged